public/components/trace_analytics/components/services/services_table.tsx

```diff
@@ -264,39 +264,41 @@ export function ServicesTable(props: ServicesTableProps) {
 
   return (
     <>
-      <EuiPanel>
-        {titleBar}
-        <EuiSpacer size="m" />
-        <EuiHorizontalRule margin="none" />
-        {!(
-          (mode === 'data_prepper' && dataPrepperIndicesExist) ||
-          (mode === 'jaeger' && jaegerIndicesExist)
-        ) ? (
-          <MissingConfigurationMessage mode={mode} />
-        ) : items?.length > 0 ? (
-          <EuiInMemoryTable
-            tableLayout="auto"
-            items={items}
-            columns={columns}
-            pagination={{
-              initialPageSize: 10,
-              pageSizeOptions: [5, 10, 15],
-            }}
-            sorting={{
-              sort: {
-                field: 'name',
-                direction: 'asc',
-              },
-            }}
-            loading={loading}
-            selection={selectionValue}
-            isSelectable={true}
-            itemId="itemId"
-          />
-        ) : (
-          <NoMatchMessage size="xl" />
-        )}
-      </EuiPanel>
+      <div style={{ padding: '0 16px' }}>
```

Comment:
let's use OUI components as much as we can

Reply:
Switched to `<EuiPage paddingSize='m'>`